FormattingStyleGuide.md

```diff
@@ -0,0 +1,43 @@
+# Formatting & Style Guide
+
+## Scope and purpose
+This document provides formatting and style conventions for publishing documents in __GitHub__ in __Markdown__ format. Some advice on document conversion and how to meet more sophisticated publishing needs is offered.
+
+## General
+- Use [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/).
+- If the workflow prescribes the document to go into BitBucket repository first, then you may find [BitBucket Markdown Syntax Guide](https://confluence.atlassian.com/bitbucketserver/markdown-syntax-guide-776639995.html) useful.
+
+## Title & headings
+- Use # (h1) for document title.
+- Use ## (h2) for top-level headings.
+- Number the headings manually. There's no heading numbering support in Markdown.
```

Comment:
Manual numbering of headings would be error prone and require unnecessary effort. I would suggest to use headings without numbering or try and find another solutions, for example, css styles.